edweiss-app/app/login/index.tsx

```diff
@@ -27,30 +51,114 @@ const Login: ApplicationRoute = () => {
 				router.replace("/");
 			} else {
 				console.log(accountRes.error);
-				setLoading(false);
+				setLoadingGoogle(false);
 			}
 		}
 	}
 
+	// Log the app Anonymously
+	async function signInAnonymous() {
+		setLoadingAnon(true);
+
+		const res = await signInAnonymously();
+
+		if (res != undefined) {
+			const accountRes = await callFunction(Auth.Functions.createAccount, { name: res.user.displayName });
+
+			if (accountRes.status == 1) {
+				router.replace("/");
+			} else {
+				console.log(accountRes.error);
+				setLoadingAnon(false);
+			}
+		}
+	}
+
+	// Get the color depending on the active state
+	function getColor(quoteNumber: number) {
+		return quoteN === quoteNumber ? 'sky' : 'surface0';
```

Comment:
Can you explain to me the meaning of the === please, just out of curiosity since it doesn't make much sense for me

Reply:
Absolutely! The `===` operator checks for strict equality in JavaScript and TypeScript, meaning it verifies that both the value and the type are identical. In this function, `quoteN === quoteNumber` checks if `quoteN` and `quoteNumber` have the exact same value and type. If they do, it returns `'sky'`; otherwise, it returns `'surface0'`.

Using `===` avoids unexpected results that might occur `if == (loose equality)` were used, which tries to convert types before comparing. 

Does this makes more sens ?

Reply:
Absolutely, thank you for the patience